### PR TITLE
CRM: Adding module exports for vars and all functions

### PR DESCRIPTION
**Pull request by coder-karen**
## Proposed changes:

* This PR adds exports for various variables throughout CRM that are referenced via 'window.variablename'. This led to the following reported error for one particular variable: https://github.com/Automattic/zero-bs-crm/issues/2878
* It also includes all remaining functions (not just those called from files outside of the file thy are declared in), as well as any other globally declared variables.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

pbhBOz-2YT-p2#comment-3740

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test on Jurassic Ninja, create a new Jurassic Ninja site with the [Beta plugin](https://github.com/Automattic/jetpack-beta) installed and active, and under Jetpack CRM on the Beta testing dashboard add this feature branch.
* The issue reported in https://github.com/Automattic/zero-bs-crm/issues/2878 should no longer exist.

### CODE REVIEW

File: projects/plugins/crm/js/ZeroBSCRM.admin.customerfilters.js [hunk 1]
```diff
@@ -185,3 +185,8 @@ function zbsJSFilterDateRangeClear() {
 	jQuery( '#zbs-crm-customerfilter-addedrange-reportrange span' ).html( '' );
 	jQuery( '#zbs-crm-customerfilter-addedrange' ).val( '' );
 }
+
+if ( typeof module !== 'undefined' ) {
+    module.exports = { zbscrmjsCustomerFilterRetrieving, zbscrmjsCustomerFilterHasRetrieved, zbscrmJS_customfilters_bindDateRange,
+		zbscrmJS_customfilters_bindAjaxButton, zbsJSFilterDateRangeCallback, zbsJSFilterDateRangeClear};
```
Comment (cleacos) [inline projects/plugins/crm/js/ZeroBSCRM.admin.customerfilters.js]:
```suggestion
		zbscrmJS_customfilters_bindAjaxButton, zbsJSFilterDateRangeCallback, zbsJSFilterDateRangeClear };
```

Comment (coder-karen) [reply projects/plugins/crm/js/ZeroBSCRM.admin.customerfilters.js]: Thanks!

**Review by cleacos — COMMENTED** [review]
I tested this PR, and it works as expected. The console logs I got, disappeared. And now I can see the missing fields `Name and Avatar` `Added` and the link on the `ID`column.

image

**Review by cleacos — APPROVED** [review]
It works as expected. The issues from these comments have disappeared:

pbhBOz-2YT-p2#comment-3740
pbhBOz-2YT-p2#comment-3744

**Pull request merged**